atom/browser/api/atom_api_web_contents.cc

```diff
@@ -757,7 +757,11 @@ void WebContents::MediaStoppedPlaying(const MediaPlayerInfo& video_type,
 }
 
 void WebContents::DidChangeThemeColor(SkColor theme_color) {
-  Emit("did-change-theme-color", atom::ToRGBHex(theme_color));
+  if (theme_color != SK_ColorTRANSPARENT) {
+    Emit("did-change-theme-color", atom::ToRGBHex(theme_color));
+  } else {
+    Emit("did-change-theme-color", nullptr_t());
```

Comment:
> nullptr will not work in this case because Emit takes a reference but not a pointer as argument. Alternatively we can overload Emit to take pointers but probably it is a overkill for this change.

I've just tested `Emit("did-change-theme-color", nullptr)` and stepped through the `nullptr` -> `v8::Null` conversion step by step to make sure it works as expected, it works, and it doesn't require any changes other than the one already in `native_mate`. I'm using VS2015.